garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -301,19 +300,19 @@ L.item_radar      = "Radar"
 L.item_radar_desc = [[
 Dies erlaubt dir Lebenszeichen zu scannen.
 Es startet automatisch beim Kauf.
-Stelle es im Radar Reiter dieses Menüs ein.]]
+Stelle es im Reiter 'Radar' dieses Menüs ein.]]
 
 L.item_disg       = "Tarnung"
 L.item_disg_desc  = [[
 Versteckt deine ID. Vermeidet außerdem, dass du die
 Person bist, die als letztes vom Opfer gesehen wurde.
 
-Schalte es im Tarnungs Reiter ein oder aus
+Schalte es im Reiter 'Tarnung' ein oder aus
 oder drücke Enter auf dem Numpad.]]
 
 -- C4
 L.c4_hint         = "Drücke {usekey} zum scharf stellen oder entschärfen."
-L.c4_no_disarm    = "Du kannst nicht das C4 eines anderen Verräters entfernen, bis dieser tot ist."
+L.c4_no_disarm    = "Du kannst das C4 eines anderen Verräters nicht entfernen, bevor dieser nicht tod ist."
```

Comment:
Either one of the Contributors change that or someone check if I made some more mistakes.
As a native german I can speak a perfect german but this doesn't exclude small mistakes. :/